Intersect.Server.Core/Core/MapInstancing/Controllers/InstanceController.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Intersect.Server.Entities;
+
+namespace Intersect.Server.Core.MapInstancing.Controllers;
+public class InstanceController
+{
+    Guid InstanceId { get; set; }
+
+    public HashSet<Player> Players { get; set; } = new HashSet<Player>();
+
+    public int PlayerCount => Players.Count;
```

Comment:
I don't think this property is really useful, it adds a method call and only saves 2 characters.

Reply:
Sure, fixed